lib/imports/api/authority-registry/server/authority-registry.js

```diff
@@ -1,35 +1,34 @@
 import {
-  insertAuthority,
-  removeAuthority,
-  storeFailedHealthChecks,
-  removeAuthorities,
-  resetFailedHealthChecksCount,
-  addDocToAuthority
+  insertAuthorityServerIntoRegistry,
+  removeAuthorityServerFromRegistry,
+  addDocToAuthorityServer,
+  removeDocFromAuthorityServer
 } from './methods';
 import { authorityRegistryColl } from './collection';
 import { WebApp } from 'meteor/webapp';
-import { HTTP } from 'meteor/http';
 import { check } from 'meteor/check';
 import { Meteor } from 'meteor/meteor';
 import ip from 'ip';
-
-const HTTPS_PORT = 443;
-const HEALTH_CHECK_RESPONSE = 'ProseMeteor Health OK';
-const FAILED_HEALTH_CHECKS_THRESHOLD = 5;
-const HEALTH_CHECK_INTERVAL_MS = 1000 * 30;   // interval on which to run health checks
+import { proseMeteorConfig } from './../../config/server/prosemeteor-config';
+import {
+  checkHealthOfServers,
+  storeHealthCheckResults,
+  removeDeadServersFromRegistry,
+  HEALTH_CHECK_RESPONSE
+} from './health-checks';
 
 /**
 *   Class that manages and communicates with the authority registry. Maintains a list of all
 *   authority servers.
 */
 export class AuthorityRegistry {
   constructor () {
-    this.ip = ip.address();
-    this.port = parseInt(process.env.PORT);
+    proseMeteorConfig.ip = ip.address();
```

Comment:
storing global config like this server's ip and port in the single config class instance